docs/quick-start.md

```diff
@@ -0,0 +1,251 @@
+   return <>
+     {/* We'll define our element presentation here */}
+   </>
+ }
+)
+```
+In `prosemirror-elements`, we’ve defined helper components to make it easy to expose inputs and data. Let’s use the `FieldWrapper` component to display an appropriate input for our `content` rich text Field:
+```js
+const pullquoteElement = createReactElementSpec(
+ pullquoteFields,
+ ({ fields, fieldValues, errors }) => {
+   return (
+     <div>
+       <FieldWrapper
+         field={fields.content}
+         errors={errors.content}
+         headingLabel="Pullquote content"
+       />
+     </div>
+   );
+ }
+);
+```
+We can do the same thing with our text `attribution` Field:
+```js
+const pullquoteElement = createReactElementSpec(
+ pullquoteFields,
+ ({ fields, fieldValues, errors }) => {
+   return (
+     <div>
+       <FieldWrapper
+         field={fields.content}
+         errors={errors.content}
+         headingLabel="Pullquote content"
+       />
+       <FieldWrapper
+         field={fields.content}
+         errors={errors.content}
+         headingLabel="Attribution"
+       />
+     </div>
+   );
+ }
+);
+```
+For our `role` Field, we can make use of another React component designed to display dropdowns, `CustomDropdownView`:
+```js
+const pullquoteElement = createReactElementSpec(
+ pullquoteFields,
+ ({ fields, fieldValues, errors }) => {
+   return (
+     <div>
+       <FieldWrapper
+         field={fields.content}
+         errors={errors.content}
+         headingLabel="Pullquote content"
+       />
+       <FieldWrapper
+         field={fields.content}
+         errors={errors.content}
+         headingLabel="Attribution"
+       />
+       <CustomDropdownView
+         field={fields.role}
+         label="Role"
+         errors={errors.role}
+         display="inline"
+       />
+     </div>
+   );
+ }
+);
+```
+And that’s it! Note that we don’t need to make use of all of these Fields in our component – we’re at liberty to choose whichever presentation we like.
+
+Our pullquote doesn’t really need it, but in other contexts we might want to work with the current values of our Fields. These are exposed in our `fieldValues` prop, so to reveal the pullquote content as we typed, we could write:
+```js
+const pullquoteElement = createReactElementSpec(
+ pullquoteFields,
+ ({ fields, fieldValues, errors }) => {
+   return (
+     <div>
+       <FieldWrapper
+         field={fields.content}
+         errors={errors.content}
+         headingLabel="Pullquote content"
+       />
+       <blockquote>
+         {fieldValues.content}
+       </blockquote>
+     </div>
+   );
+ }
+);
+```
+
+## Where to keep our element
+By convention, we’ve defined and kept our elements in `src/elements`. Let’s add our pullquote there, in `src/elements/example-pullquote/ExamplePullquote.tsx`, and export the element we’ve just created:
+```js
+export const pullquoteElement = createReactElementSpec(
+  // …
+```
+
+# Creating the Element Form
```

Comment:
Is this section still necessary? We've introduced these concepts further up.

Reply:
Of course we have done this in 1 file